utils/past_ci_versions.py

```diff
@@ -0,0 +1,137 @@
+import argparse
+import os
+
+
+past_versions_testing = {
+    "pytorch": {
+        "1.10": {
+            "torch": "1.10.2",
+            "torchvision": "0.11.3",
+            "torchaudio": "0.10.2",
+            "python": 3.9,
```

Comment:
This is the highest python version for that PyTorch version. Not really important